homeassistant/bootstrap.py

```diff
@@ -539,12 +552,21 @@ async def _async_set_up_integrations(
             async with hass.timeout.async_timeout(
                 STAGE_2_TIMEOUT, cool_down=COOLDOWN_TIME
             ):
-                await async_setup_multi_components(
-                    hass, stage_2_domains, config, setup_started
-                )
+                await async_setup_multi_components(hass, stage_2_domains, config)
         except asyncio.TimeoutError:
             _LOGGER.warning("Setup timed out for stage 2 - moving forward")
 
+    log_task.cancel()
+    _LOGGER.info(
```

Comment:
I'm not sure if this is a real addition, maybe as debug level?

Especially as each integration has already reported it individually.

![image](https://user-images.githubusercontent.com/195327/113059156-64b7bf00-91af-11eb-8ee3-c528fea560d8.png)


![image](https://user-images.githubusercontent.com/195327/113059208-74cf9e80-91af-11eb-9391-a41491b97be2.png)

Reply:
Switched to debug